scene/gui/tabs.cpp

```diff
@@ -579,7 +579,7 @@ void Tabs::_update_cache() {
 					slen -= tabs[i].icon->get_width();
 					slen -= get_constant("hseparation");
 				}
-				if (cb_displaypolicy == CLOSE_BUTTON_SHOW_ALWAYS || (cb_displaypolicy == CLOSE_BUTTON_SHOW_ACTIVE_ONLY && i == current)) {
+				if (cb_displaypolicy == CLOSE_BUTTON_SHOW_ALWAYS || (cb_displaypolicy == CLOSE_BUTTON_SHOW_ACTIVE_ONLY)) {
```

Comment:
That might introduce a bug, the previous condition was always false, but now it's going to add a close button on all tabs if the policy is `CLOSE_BUTTON_SHOW_ACTIVE_ONLY`. CC @StraToN